clang/include/indexstore/indexstore.h

```diff
@@ -299,6 +299,8 @@ INDEXSTORE_OPTIONS(uint64_t, indexstore_symbol_property_t) {
   INDEXSTORE_SYMBOL_PROPERTY_GKINSPECTABLE                    = 1 << 6,
   INDEXSTORE_SYMBOL_PROPERTY_LOCAL                            = 1 << 7,
   INDEXSTORE_SYMBOL_PROPERTY_PROTOCOL_INTERFACE               = 1 << 8,
+
+  INDEXSTORE_SYMBOL_PROPERTY_ASYNC                            = 1 << 16,
```

Comment:
Use `INDEXSTORE_SYMBOL_PROPERTY_SWIFTASYNC` to make it clear this is Swift specific.

Reply:
Renamed to `INDEXSTORE_SYMBOL_PROPERTY_SWIFT_ASYNC`, let me know if you feel strongly about `SWIFTASYNC` though